static/app/views/settings/organizationIntegrations/integrationContext.tsx

```diff
@@ -0,0 +1,22 @@
+import {createContext} from 'react';
+
+import type {IntegrationProvider, IntegrationType} from 'sentry/types/integrations';
+
+export type IntegrationContextProps = {
+  analyticsParams: {
+    already_installed: boolean;
+    view:
+      | 'integrations_directory_integration_detail'
+      | 'integrations_directory'
+      | 'onboarding'
+      | 'project_creation';
```

Comment:
i like how exact this is.. but i wonder if we could also just use `string` here to make it a bit more reusable for the future?

Reply:
Unfortunately we can't use `string` here because the analyticsParams type is defined by [AddIntegration](https://github.com/getsentry/sentry/blob/master/static/app/views/settings/organizationIntegrations/addIntegration.tsx#L22-L29)